app/poros/movie_data.rb

```diff
@@ -0,0 +1,29 @@
+class MovieData
```

Comment:
Clever use of class methods here to pull out the correct data in one method. Seems like the convergence of the different API endpoints would be tricky to figure out, so nice job on this solution!